packages/js/components/src/sortable/sortable.tsx

```diff
@@ -265,7 +266,7 @@ export const Sortable = ( {
 						<SortableItem
 							key={ child.key || index }
 							className={ itemClasses }
-							id={ index }
+							id={ `${ index }${ v4() }` }
```

Comment:
Optional, but I think we could remove `index` or add a `-` here if we need to keep the index for debugging.

Reply:
Sounds good to me! I added the `-` between the item `index` and the `id` in the commit [248e3e1](https://github.com/woocommerce/woocommerce/pull/36262/commits/248e3e1eb07754af622a1575ae5cb614de45f8a9)